examples/gnrc_border_router/Makefile

```
@@ -27,6 +27,19 @@ USEMODULE += ps
 # Optionally use DHCPv6 instead of UHCP
 USE_DHCPV6 ?= 0
 
+# Default to using ethos to provide the uplink when not on native
+USE_ETHOS ?= 1
+# Optionally use SLIP instead of ethos when not on native
+# (added to USEMODULE in Makefile.board.dep)
+USE_SLIP  ?= 0
+# Optionally use esp_wifi & esp_now with an esp8266 or esp32
+USE_ESP   ?= 0
```

Why restrict this to ESP-WiFi?
```suggestion
USE_WIFI   ?= 0
```

The doc should also use specify that the WiFi interface is the upstream interface.

---

Well `esp_now` is an esp specific technology.
But it's easy enough to swap `esp_now` with e.g. a 802.15.4 radio driver.

---

Yeah, I don't see a reason to enforce `esp_now` ;-).